### ISSUE TITLE
CC0 license problems: go to CC-BY ?

### ISSUE THREAD

**Issue by jcolomb** [OpenScienceMOOC/Main#27]
I was today in a very interesting talk about licenses.
In brief, there is 2 main legal system, one for UK/USA, and one for Europe (and their respective former colonies). The European one has no copyrights law, but author's rights. That means that CC0 is irrelevant in Europe: you cannot waive attribution necessity (as well as change the title). It also means that you legally have to give attribution to CC0 licensed material if you use in Europe. To simplify, CC0 becomes CC-BY if the author or the user is in Europe (but that very few people will realise it). 
That means that the CC0 label is misleading in most cases.

I therefore propose to change the license to CC-BY, and add a note about why we changed.

**Comment by dannycolin** [comment]
> To simplify, CC0 becomes CC-BY if the author or the user is in Europe (but that very few people will realise it).

TIL. 

> I therefore propose to change the license to CC-BY, and add a note about why we changed.

Since the discussion to relicense is open (it was on my todo list :)) could we also consider adding the Share-Alike attribute. This would ensure that any derivative of our work will stay freely available.

**Comment by Protohedgehog** [comment]
Hey @dannycolin yes that should be totally fine, I think :)

**Comment by bmkramer** [comment]
Sorry to jump in out of the blue sky :-) I know this is an seminal
discussion point with clear parallels in the open software community.
I am quite firmly in the camp of CC-BY over CC-BY-SA, as the former allows
more wider use, while still ensuring that the content itself cannot be
locked in (even though yes, derivatives might be)

kind regards, '
Bianca




Op ma 25 mrt. 2019 om 15:17 schreef Jon Tennant <notifications@github.com>:

**Comment by dannycolin** [comment]
> I am quite firmly in the camp of CC-BY over CC-BY-SA, as the former allows more wider use

@bmkramer do you have some examples of use case where CC-BY-SA will narrow the use of our material?

**Comment by bmkramer** [comment]
For one thing, just like we cannot use any material from Wikipedia (which
is CC-BY-SA) if we license our course CC-BY, the same would go for any
other parties that license CC-BY being excluded from using our material if
we license CC-BY-SA.

Also, any other, more restrictive licensing that people might want to use
would prohibit the inclusion of CC-BY-SA material. While the argument could
be made that that is exactly the point of SA, it does limit i.e. inclusion
in courses that are not openly licensed. I don't have a real-world example
at hand, but I could image re-use on university webpages that might not
even employ a license, or an all-rights-reseved book  ;-)

Basically, the way I see it that CC-BY is a way to enable the most wide
usage (irrespective of downstream licensing) and CC-BY-SA as a way to
ensure downstream open sharing. Both have their advantages....


Op ma 25 mrt. 2019 om 16:12 schreef Danny Colin <notifications@github.com>:

**Comment by gedankenstuecke** [comment]
> For one thing, just like we cannot use any material from Wikipedia (which is CC-BY-SA) if we license our course CC-BY, the same would go for any other parties that license CC-BY being excluded from using our material if we license CC BY-SA.

Doesn't it depend a bit on whether we clearly license the CC BY-SA bits used inside our materials as such? E.g. I can use CC BY-SA images in my CC BY blog when clearly giving the CC BY-SA license for that specific image?

**Comment by dannycolin** [comment]
> Doesn't it depend a bit on whether we clearly license the CC BY-SA bits used inside our materials as such? E.g. I can use CC BY-SA images in my CC BY blog when clearly giving the CC BY-SA license for that specific image?

Exactly. You can use CC-BY-SA materials in a copyrighted work (e.g. a book) because your book isn't considered a derivative work. What you can't do is modifying the image and use a CC-BY license. 

Further, if someone made a modification of our work, s/he only need to license the modification under the same license. Again, if you modify a CC-BY-SA image and use it in your copyrighted book, you only need to keep the same license on that new image not on the whole book.

**Comment by jcolomb** [comment]
I am against CC-BY-SA. As a principle we want to ensure wider reuse, SA is a restriction. 

Even if in practice it would not make a big difference, it does in term of principle. real Open access is CC-BY, without any restriction.

**Comment by pcmasuzzo** [comment]
While I use CC-BY-SA on daily base (and I do not agree that "real OA is CC-BY"), I believe we should stick to CC-BY, as SA might indeed restrict re-use.

My 2 cents.

**Comment by dannycolin** [comment]
> As a principle we want to ensure wider reuse, SA is a restriction.

One can argue that someone being able to re-use and keep their modifications closed limit a wider reuse of the collective work.

If we go with the CC-BY, it would be nice if we could at least have a something on the website saying that we encourage people to give back if possible (whatever the open license they use for their derivative work).

**Comment by jcolomb** [comment]
"something on the website saying that we encourage people to give back if possible" 

Isn't the whole MOOC about just that ? ;)

**Comment by dannycolin** [comment]
@jcolomb sure but I was referring also referring to the other platforms where the license information is. (e.g GitHub). I mean something that said why we use a CC license and why we encourage people to do so.

**Comment by Respoda** [comment]
> 
> 
> I was today in a very interesting talk about licenses.
> In brief, there is 2 main legal system, one for UK/USA, and one for Europe (and their respective former colonies). The European one has no copyrights law, but author's rights. That means that CC0 is irrelevant in Europe: you cannot waive attribution necessity (as well as change the title). It also means that you legally have to give attribution to CC0 licensed material if you use in Europe. To simplify, CC0 becomes CC-BY if the author or the user is in Europe (but that very few people will realise it).
> That means that the CC0 label is misleading in most cases.
> 
> I therefore propose to change the license to CC-BY, and add a note about why we changed.

Afaik this is not true, or at least no longer true. The European Commission has adopted both CC0 and CC-BY, and they are regarded the same as anywhere else in the world. Took me hours to try and confirm your statement, and whereas I can find no confirmation whatsoever regarding cc0 ever having been considered cc-by in Europe, I can find official confirmation that the EC has adopted CC0 as a legal means for sharing and redistribution under the creative commons rules, as well as CC-BY and CC-BY-SA.

It's statements like this that only add confusion to what is already a bloody mess...

**Comment by jcolomb** [comment]
It is true, discussed with CC specialist the other day and they did confirm it. As you would read here:
http://publications.jrc.ec.europa.eu/repository/bitstream/JRC115947/kjna29685enn_1.pdf

"However, because dedicating works to the public domain is not permissible in many jurisdictions, CC0 includes a public licence fall-back clause to address situations where the applicable law does not permit surrendering or waiving copyright. In such cases, the tool becomes a very permissive licence, while the authors declare that they will not exercise any remaining copyright in the work or assert any associated claims, hence giving the creator a way to waive all their copyright and related rights in their works to the fullest extent allowed by law."

"Additionally, the following questions persist in relation to CC0:
 it is uncertain whether the extensive waiver would be held valid in some jurisdictions, especially in the continental "droit d'auteur" jurisdictions, such as in Belgium, as it could be seen by some as contradicting some of the basic principles of the inalienable "droit d'auteur";"

"CC0 could be used for marking "documents in the public domain (e.g. where IPR protection has expired or in jurisdictions where official documents are exempt from copyright protection by law)".63 CC0 could also be considered as an option for the open dissemination of datasets.
"
The "droit d'auteur" is the urheberrecht in Germany and exist in most European countries. 

There is no "droit d'auteur" for dataset, though, and that explain why the EC can use it for data and metadata.
